packages/vue-renderer/src/loading.html

```diff
@@ -0,0 +1,280 @@
+            <path d="M235.702 175.491L172.321 62.216c-.655-1.191-4.313-7.216-10.68-7.216-2.868 0-6.977 1.237-10.32 7.193L143 75.706v26.104l18.709-32.31 62.704 111.626h-23.845c.305 1.846.134 3.74-.496 5.498a7.06 7.06 0 0 1-.497 1.122l-.203.413c-3.207 5.543-10.139 5.841-11.494 5.841h37.302c1.378 0 8.287-.298 11.493-5.841 1.423-2.52 2.439-6.758-.97-12.668z" fill="#108775"/>
+            <path d="M201.608 189.07l.21-.418c.206-.364.378-.745.515-1.139a10.94 10.94 0 0 0 .515-5.58 16.938 16.938 0 0 0-2.152-5.72l-49.733-87.006L143.5 76h-.136l-7.552 13.207-49.71 87.006a17.534 17.534 0 0 0-1.917 5.72c-.4 2.21-.148 4.486.725 6.557.13.31.278.613.444.906 1.497 2.558 4.677 5.604 11.691 5.604h92.592c1.473 0 8.651-.302 11.971-5.93zm-58.244-86.657l45.455 79.52H97.934l45.43-79.52z" fill="#2F495E" fill-rule="nonzero"/>
+          </g>
+        </svg>
+      </transition>
+    </div>
+    <div class="row placeholder" v-if="!bundles.length">
+      <transition appear>
+        <h3>Loading...</h3>
+      </transition>
+    </div>
+    <transition-group>
+      <div class="row" v-for="bundle of bundles" :key="bundle">
+        <h3>{{ bundle | capitalize}} bundle</h3>
+        <div class="progress_bar_container">
+          <div class="progress_bar" :class="bundle" :style="{ width: `${states[bundle].progress}%` }"></div>
+        </div>
+        <h4>{{ states[bundle].status }}</h4>
+      </div>
+    </transition-group>
+  </div>
+
+  <script>
+  new Vue({
+    el: '#loading_screen',
+    data () {
+      return {
+        finished: false,
+        bundles: [],
+        states: {
+          client: {
+            progress: 0,
+            status: 'Bundling...'
+          },
+          server: {
+            progress: 0,
+            status: 'Bundling...'
+          },
+          modern: {
+            progress: 0,
+            status: 'Bundling...'
+          }
+        }
+      }
+    },
+    created () {
+      this.ws = null
+      this.autoReconnectInterval = 1000 // 1 second
+      this.connected = false
+      this.connect()
+    },
+    methods: {
+      connect () {
+        this.url = `ws://${location.hostname}:${location.port}/_webpack/ui/ws`
+        console.log('WebSocket(client): connecting...')
+        this.ws = new WebSocket(this.url)
+        this.ws.onopen = () => {
+          console.log('WebSocket(client): connected')
+          this.connected = true
+        }
+        this.ws.onclose = (e) => {
+          this.connected = false
+          switch (e.code){
+            case 1000: // CLOSE_NORMAL
+            case 1005:
+              console.log('WebSocket(client): closed')
+              break;
+            default: // Abnormal closure
```

Comment:
```suggestion
            default: // Unkown error
```

Does that fit better? :thinking:

Reply:
done